src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,60 @@
+.calendar {
+  $root: &;
+
+  display: flex;
+  flex-wrap: wrap;
+
+  width: ($day-box-size * 7) + ($gap-size * 6) + (2 * $padding-horizontal);
+  padding: 30px $padding-horizontal;
+
+  gap: $gap-size;
+
+  &__day {
+    display: flex;
+
+    width: $day-box-size;
+    height: $day-box-size;
+
+    border: 1px solid #000;
+    background-color: #eee;
+
+    transition: all 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+
+      transform: translateY(-20px);
+    }
+
+    @for $date from 1 through 31 {
+      &:nth-child(#{$date}) {
+        align-items: center;
+        justify-content: center;
+
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
```

Comment:
not critical, but it would be cleaner to move all these to `&__day`
and just leave the variable part inside `@for`
`&:nth-child(#{$date})::before {`